samples/dotnet/EdgeX509AuthDownstreamDevice/Program.cs

```diff
@@ -218,17 +241,17 @@ static void Main()
                 }
             }
 
-            TransportType transport = GetTransport(ClientTransportType);
-
             InstallCACert();
 
             Console.WriteLine("Creating device client using identity certificate...\n");
 
             var (cert, certChain) = GetClientCertificateAndChainFromFile(DeviceIdentityCertPath, DeviceIdentityPrivateKeyPath);
             InstallChainCertificates(certChain);
-            var auth = new DeviceAuthenticationWithX509Certificate(DownstreamDeviceId, cert);
 
-            DeviceClient deviceClient = DeviceClient.Create(IothubHostname, IotEdgeGatewayHostname, auth, transport);
+            ITransportSettings[] transportSettings = GetTransport(ClientTransportType);
+
+            var auth = new DeviceAuthenticationWithX509Certificate(DownstreamDeviceId, cert);
```

Comment:
nit: use explicit type instead of var; as you can't tell what type is from right side.

Reply:
My understanding was that var was ok since auth is of type DeviceAuthenticationWithX509Certificate. Am I misunderstanding something?

Reply:
Yeah I am not seeing an issue here.